src/components/Navrail/Navrail.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  Search,
+  LayoutGrid,
+  CircleDollarSign,
+  Wallet,
+  Zap,
+  Gem,
+  Settings2,
+  CircleHelp,
+  Gift,
+} from 'lucide-react';
+import React, { FC } from 'react';
+import {
+  Sidebar,
+  SidebarContent,
+  SidebarFooter,
+  SidebarGroup,
+  SidebarGroupContent,
+  SidebarGroupLabel,
+  SidebarHeader,
+  SidebarMenu,
+  SidebarMenuButton,
+  SidebarMenuItem,
+  SidebarSeparator,
+  SidebarTrigger,
+  useSidebar,
+} from '@/components/ui/sidebar';
+
+import styles from './Navrail.module.scss';
+
+const menuOne = [
```

Comment:
better to keep the naming as the same as the css classes they're used with
```suggestion
const Menu = [
```